db/migration/createDB.js

```diff
@@ -0,0 +1,4 @@
+import User from '../../api/user/repository/userRepository.js';
+
+await User.sync({force: true});
```

Comment:
for migration scripts, we need to have two options - up and down.
up - apply changes in the DB
down - undo changes in the DB

You need both migrations and seeds in your project. 
migration - creates or updates tables in the project
seed - fill the table with values

please use this guide to create Migrations and Seeds in your project
[guide link](https://reyhanhamidi.medium.com/sequelize-automatic-database-migration-and-seeding-on-heroku-fb88cf09573b)
[docs link](https://sequelize.org/master/manual/migrations.html#the-sequelizerc-file)